Makefile

```diff
@@ -0,0 +1,67 @@
+.PHONY: all clone-quickstart install-quickstart start-quickstart install-openmct-yamcs sanity-test build-example test-getopensource test-e2e clean
+
+all: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example test-e2e
+
+start: clone-quickstart install-quickstart install-openmct-yamcs sanity-test build-example start-openmct
+
+clone-quickstart:
+	@echo "Running target: clone-quickstart"
+	@echo "Current working directory: $(shell pwd)"
+	if [ ! -d "quickstart" ]; then \
+		git clone https://github.com/yamcs/quickstart; \
+	else \
+		echo "Directory 'quickstart' already exists."; \
+	fi
+
+install-quickstart:
+	@echo "Running target: install-quickstart"
+	@cd quickstart/docker && $(MAKE) wait-for-sent
+
+start-quickstart:
+	@echo "Running target: start-quickstart"
+	@cd quickstart/docker && $(MAKE) all
+
+restart-quickstart:
```

Comment:
restart vs reset below, should they be consistent?

Reply:
ohhh